src/dialogs/mqtt-device-debug-info-dialog/dialog-mqtt-device-debug-info.ts

```diff
@@ -0,0 +1,227 @@
+import {
+  LitElement,
+  css,
+  html,
+  CSSResult,
+  TemplateResult,
+  customElement,
+  property,
+} from "lit-element";
+import "../../components/ha-dialog";
+import "../../components/ha-switch";
+import { computeDeviceName } from "../../data/device_registry";
+import { computeStateName } from "../../common/entity/compute_state_name";
+import { haStyleDialog } from "../../resources/styles";
+// tslint:disable-next-line: no-duplicate-imports
+import { HaSwitch } from "../../components/ha-switch";
+import { HomeAssistant } from "../../types";
+import { MQTTDeviceDebugInfoDialogParams } from "./show-dialog-mqtt-device-debug-info";
+import { MQTTDeviceDebugInfo, fetchMQTTDebugInfo } from "../../data/mqtt";
+import "./mqtt-payload";
+
+@customElement("dialog-mqtt-device-debug-info")
+class DialogMQTTDeviceDebugInfo extends LitElement {
+  public hass!: HomeAssistant;
+  @property() private _params?: MQTTDeviceDebugInfoDialogParams;
+  @property() private _debugInfo?: MQTTDeviceDebugInfo;
+  @property() private _showAsYaml: boolean = false;
+  @property() private _showDeserialized: boolean = false;
+
+  public async showDialog(
+    params: MQTTDeviceDebugInfoDialogParams
+  ): Promise<void> {
+    this._params = params;
+    fetchMQTTDebugInfo(this.hass, params.device.id).then((results) => {
+      this._debugInfo = results;
+    });
+    await this.updateComplete;
```

Comment:
Do we need to await this if nothing is happening after it?

Reply:
We should probably resize after right?
```ts
fireEvent(this._dialog as HTMLElement, "iron-resize");
```

Reply:
This isn't an `paper-dialog` but an `mwc-dialog`